src/ucp/api/ucp_compat.h

```diff
@@ -12,6 +12,17 @@
 #include <ucp/api/ucp_def.h>
 
 
+ /**
+ * @ingroup UCP_COMM
+ * @deprecated Replaced by @ref ucp_ep_params_t::err_handler_cb and
+ *            @ref ucp_ep_params_t::user_data.
+ */
+typedef struct ucp_err_handler {
```

Comment:
i would not keep this structure in ucp_compat.h, since it can't be used without ucp_ep_params_t anyway